--- ironrdp-connector-0.2.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ ironrdp-connector-0.2.0/Cargo.toml	2024-11-27T14:20:25.799924+00:00
@@ -67,14 +67,14 @@
 version = "0.5.0"
 
 [dependencies.picky-asn1-x509]
-version = "0.13.0"
+version = "0.14.0"
 
 [dependencies.rand_core]
 version = "0.6"
 features = ["std"]
 
 [dependencies.sspi]
-version = "0.14"
+version = "0.15"
 
 [dependencies.tracing]
 version = "0.1"
