--- picky-7.0.0-rc.12/Cargo.toml	1970-01-01T00:00:01+00:00
+++ picky-7.0.0-rc.12/Cargo.toml	2025-02-03T09:04:23.329668+00:00
@@ -59,23 +59,11 @@
 features = ["std"]
 optional = true
 
-[dependencies.argon2]
-version = "0.5"
-optional = true
-
 [dependencies.base64]
 version = "0.22"
 
-[dependencies.bcrypt-pbkdf]
-version = "0.10"
-optional = true
-
 [dependencies.byteorder]
 version = "1.5"
-optional = true
-
-[dependencies.cab]
-version = "0.6"
 optional = true
 
 [dependencies.cbc]
@@ -88,10 +76,6 @@
 features = ["clock"]
 optional = true
 default-features = false
-
-[dependencies.ctr]
-version = "0.9"
-optional = true
 
 [dependencies.des]
 version = "0.8"
@@ -118,10 +102,6 @@
 version = "1.1"
 optional = true
 
-[dependencies.lexical-sort]
-version = "0.3"
-optional = true
-
 [dependencies.md5]
 version = "0.10"
 features = ["oid"]
@@ -242,16 +222,6 @@
 chrono_conversion = [
     "dep:chrono",
     "picky-asn1/chrono_conversion",
-]
-ctl = [
-    "picky-asn1-x509/ctl",
-    "pkcs7",
-    "chrono_conversion",
-]
-ctl_http_fetch = [
-    "dep:reqwest",
-    "dep:cab",
-    "ctl",
 ]
 default = [
     "x509",
@@ -283,19 +253,6 @@
     "x509",
     "picky-asn1-x509/pkcs7",
 ]
-putty = [
-    "dep:argon2",
-    "dep:hmac",
-    "ssh",
-]
-ssh = [
-    "dep:byteorder",
-    "dep:aes",
-    "dep:ctr",
-    "dep:cbc",
-    "dep:bcrypt-pbkdf",
-    "dep:lexical-sort",
-]
 time_conversion = [
     "dep:time",
     "picky-asn1/time_conversion",
